app/(docs)/dcs/api/s3/s3-compatibility/storj-vs-self-managed-encryption-s3-compatibility-differences/page.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Storj Managed vs. Self-Managed Encryption S3 Compatibility Differences
+docId: oozaauQueuq9hgeac4Ze
+weight: 1
+---
+
+## Listing Endpoints Behavior
+
+The behavior of listing endpoints depends on the encryption model configured for your project.
+
+### Self-Managed Encryption
+When using **Self-Managed Encryption**, bucket paths are end-to-end encrypted. This prevents the listing endpoint from seeing the plaintext paths, which affects sorting and filtering.
```

Comment:
What do you think about using only "object key" terminology instead of bucket path in this doc?  I find object key a little less confusing.